docs/blog/2023/new-publishing-framework/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: New publishing framework
-description: WebSight CMS and WebSight DXP are two different products, and we want to ensure each has its unique role. CMS is designed to provide the best authoring experience, while DXP will concentrate on distributing the experiences and delivering them efficiently. We want CMS to be usable on its own. To make this possible, we plan to include response-reply publishing and move SSG module to DXP.
+description: Read more on the new publishing framework for WebSight CMS and why we need it.
```

Comment:
'Read more on the new publishing framework for WebSight CMS and why we need it.'
what about:
'Read about a new publishing framework that we are going to introduce for WebSight CMS and find out why we need it.'